src/rsz/src/SizeDownMove.cc

```diff
@@ -123,7 +123,8 @@ bool SizeDownMove::doMove(const Path* drvr_path,
     LibertyCell* load_cell = load_port->libertyCell();
     Instance* load_inst = network_->instance(load_pin);
 
-    if (resizer_->dontTouch(load_inst)) {
+    if (resizer_->dontTouch(load_inst)
+        || !resizer_->isLogicStdCell(load_inst)) {
```

Comment:
I see similar checks in various places.  Perhaps you could do a refactor similar to okToBufferNet like okToResizeInstance ?

Reply:
Do you want to do this here or in another?